README.md

```diff
@@ -1,6 +1,11 @@
 # FFC docker base
 
-This repository contains the docker files that build parent container images for FFC projects to extend. Each sub-folder contains the source files required to build a single image.
+This repository contains the docker files that build parent container images for FFC projects to extend. Each sub-folder contains the source files required to build a single image. There is also an examples folder that contains example Dockerfiles for various services. These can be used to build a Dockerfile for your service which uses the parent images created by this project.
+
+## Example files
+
+Dockerfile.web - This is an example web project, that requires a build step to create some static files that are used by the web front end.
+Dockerfile.service - This is an example project that doesn't expose any external ports (a message based service). There is also no build step in this dockerfile.
```

Comment:
could do with a line break here as they are rendered on the same line in the readme.
Also `dockerfile.web` renders as a url, so might need some special formatting to prevent it.

Reply:
Fixed.